### ISSUE TITLE
Step 02 problem

### ISSUE THREAD

**Issue by HN-Jiang** [yumorishita/LiCSBAS#254]
Dear Yumorishita,
Hi, I run in to a problem with step 02. I have 204 interferograms made by Gamma. And when I try to put them into licsbas, it have problem with step02:
![图片](https://github.com/yumorishita/LiCSBAS/assets/53942586/39bea41e-4e1b-4cea-9452-2354c4b26bec)
It always stop at 190/204th or 200/204th, and stop there without any error. When I check GEOCml1 folder, i found it can read 202 interoferograms, i tried to delate the other 2 interferograms and rerun step 02 but this time it only ready 200. Do you konw what's the probllem? 
[202306051137batch_LiCSBAS_02_02.log](https://github.com/yumorishita/LiCSBAS/files/11652872/202306051137batch_LiCSBAS_02_02.log)

ps: I tried again this time it read 195 of the 204 interferograms and stop here.

**Comment by HN-Jiang** [comment]
Actually last night I changed multilook factor from 1 to 5 then it works. All interferograms can be read.

BTW, you mentioned n_para which is use to limit the number of paralell or cpu cores. I also want to report this error or bug. In my and some colleagues use, I think this parameter didn't work. When the data is too large, even we set n_para to 1, it still call all the computing power of the server.
